openvdb/openvdb/tree/NodeManager.h

```diff
@@ -3,13 +3,12 @@
 
 /// @file tree/NodeManager.h
 ///
-/// @author Ken Museth
+/// @authors Ken Museth, Dan Bailey
```

Comment:
Do we have any guidelines on this? I've seen some project use `Modified by:`. Fine by me, I just haven't been changing authors of existing headers in PRs (filter/morphology etc)

Reply:
Author tags are a bit of a weird thing anyways, I mean, that is what version control is for :>  It's fine by me to leave it unspecified.

Reply:
Perhaps a topic for a TSC meeting? There is a small benefit to me at least in knowing who to direct a question to without going through version control history. Up until ASWF adoption, Git commits were typically from the OpenVDB DWA GitHub account, whereas these author comments are often more informative. I sort of go by the assumption that everyone that's made a substantial contribution should be listed as primary authors for quick reference, but we can discuss changing this...